<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>256</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>256</width>
    <height>256</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>256</width>
    <height>256</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Music</string>
  </property>
  <property name="windowIcon">
   <iconset theme="multimedia-player">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imgCover">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>256</width>
      <height>256</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="pixmap">
     <pixmap resource="music_resources.qrc">:/nocover.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblTrack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>256</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">Unknown Track</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblAlbum">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>256</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">Unknown Ablum</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblArtist">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>256</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">Unknown Artist</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btnPlayPause">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>220</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="icon">
     <iconset theme="media-playback-pause">
      <normaloff>.</normaloff>.</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="positionSlider">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>200</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>226</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="btnRepeat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>222</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="media-repeat-all">
      <normaloff>.</normaloff>.</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="btnNext">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>222</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="media-skip-forward">
      <normaloff>.</normaloff>.</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="btnPrevious">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>222</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="media-skip-backward">
      <normaloff>.</normaloff>.</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="music_resources.qrc"/>
 </resources>
 <connections/>
</ui>
